### PR TITLE
generate separate map files rather than dump to stdout

### PR DESCRIPTION
**Pull request by edquist**
I've been asked to have the manual-mapfile-from-gumsdb.py script generate separate map files rather than dump everything to stdout.

New output:

```
root@el6-gumsx ~ # ./manual-mapfile-from-gumsdb.py 
Writing ban-mapfile ...
Writing grid-mapfile ...
Writing voms-mapfile ...
root@el6-gumsx ~ # cat ban-mapfile
# /etc/grid-security/ban-mapfile
# http://opensciencegrid.github.io/docs/security/lcmaps-voms-authentication/#banning-users

"/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=People/CN=Alpha Flight 7727"
"/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=People/CN=Dancing Zorba 4528"
"/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=People/CN=Freddie Redonschnider 1234"
root@el6-gumsx ~ # cat grid-mapfile
# /etc/grid-security/grid-mapfile
# http://opensciencegrid.github.io/docs/security/lcmaps-voms-authentication/#mapping-users

"/DC=com/DC=example/OU=People/CN=Example User 12345" test
"/DC=org/DC=doegrids/OU=People/CN=Janet Doe 123456" osg
"/DC=org/DC=doegrids/OU=People/CN=Jane Doe 12345" osg
"/DC=com/DC=example/OU=People/CN=Example User 12345" osg
"/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=People/CN=Carl Edquist 1013" osg
root@el6-gumsx ~ # cat voms-mapfile
# /etc/grid-security/voms-mapfile
# http://opensciencegrid.github.io/docs/security/lcmaps-voms-authentication/#mapping-vos

"/osg/testgroup/Role=somerole/*" sam
```

### CODE REVIEW

**Comment by brianhlin** [comment]
That's reasonable enough but why would they have those files on a GUMS host?

**Comment by matyasselmeci** [comment]
They probably won't have the other two, but maybe they made a grid-mapfile for testing at one point, who knows. I'm kinda strict about not clobbering user files.

**Comment by edquist** [comment]
Ok, I've updated it to write the files with a `.additions` suffix.

File: bin/manual-mapfile-from-gumsdb.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def get_vug_pattern(vug):
 
 print "Writing %s ..." % GRID_MAPFILE
 with open(GRID_MAPFILE, 'w') as f:
-    print >>f, "# /etc/grid-security/%s" % GRID_MAPFILE
+    print >>f, "# /etc/grid-security/grid-mapfile"
```
Comment (matyasselmeci) [inline bin/manual-mapfile-from-gumsdb.py]: Did you forget the "Add to" comment like you added for BAN_MAPFILE above?

File: bin/manual-mapfile-from-gumsdb.py [hunk 2]
```diff
@@ -322,7 +322,7 @@ def get_vug_pattern(vug):
 if custom_voms_maps:
     print "Writing %s ..." % VOMS_MAPFILE
     with open(VOMS_MAPFILE, 'w') as f:
-        print >>f, "# /etc/grid-security/%s" % VOMS_MAPFILE
+        print >>f, "# /etc/grid-security/voms-mapfile"
```
Comment (matyasselmeci) [inline bin/manual-mapfile-from-gumsdb.py]: Same thing re. "Add to"

File: bin/manual-mapfile-from-gumsdb.py (hunk 1 above)
Comment (edquist) [reply bin/manual-mapfile-from-gumsdb.py]: Derp, yes.

**Comment by edquist** [comment]
Updated...

**Comment by edquist** [comment]
Tagged `mapfile-generator-0.2`

Raw download link:
https://raw.githubusercontent.com/opensciencegrid/osg-vo-config/mapfile-generator-0.2/bin/manual-mapfile-from-gumsdb.py

**Pull request merged**